### ISSUE TITLE
Picture Set error

### ISSUE THREAD

**Issue by EMerchant90** [rhiestan/Regard3D#41]
Hi after trying to add a data set, custom or the castle, when I try to click okay after uploading images I get the following error "Please wait until all image infos have been listed". 

What's the best way around this/fixing this?

Thank you,

**Comment by rhiestan** [comment]
This message occurs if the parsing of all image files is not yet completed. After a few seconds it should be possible to close the dialog.

**Comment by philgough** [comment]
I'm having this issue as well. The data seems to have been read and after a few minutes the dialogue still appears. 

![Screen Shot 2021-08-24 at 9 42 40 am](https://user-images.githubusercontent.com/1566694/130533341-14f09e24-10b5-4ec7-91c5-c4c682e5c9bd.png)

**Comment by philgough** [comment]
follow-up: I tried with a picture set from online, and it seems to work.

**Comment by ThirumlaDevi** [comment]
I am facing the same issue @philgough, any suggestions?

**Comment by rhiestan** [comment]
It looks like Regard3D has problems with some kind of pictures. You could try to change/delete the EXIF infos, or resize pictures.

**Comment by ThirumlaDevi** [comment]
@rhiestan yes hand choosing images with certain sizes and resizing the other worked. Thank you so much for looking out.

**Comment by DanStach** [comment]
also facing the same issue :(
